src/scores/categorical/multicategorical_impl.py

```diff
@@ -216,8 +219,10 @@ def _single_category_score(
     # Bring back NaNs
     condition1 = condition1.where(~np.isnan(fcst))
     condition1 = condition1.where(~np.isnan(obs))
+    condition1 = condition1.where(~np.isnan(categorical_threshold))
```

Comment:
Just a note to check this extra line doesn't have any problems running when categorical_threshold is a sequence of floats. Presumably this would be covered by the pre-existing tests so may not actually need more tests, but it should be confirmed.

Reply:
There are tests for sequences of floats